book/listings/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2018"
 once_cell = "1"
 log = "0.4"
 pretty_env_logger = "0.4"
+futures = "0.3"
```

Comment:
Where is it used?

Reply:
Ah nowhere I experimented and forgot to remove it

Reply:
Btw @sdroege this is no where near to being review ready so please don't spend your time on it yet 😊